Source/Core/Core/Slippi/SlippiPremadeText.h

```diff
@@ -1,5 +1,6 @@
-#pragma once
+﻿#pragma once
 
+#include "../../Common/StringUtil.h"
```

Comment:
is a relative path needed? dont remember seeing this anywhere else this is included uses `#include "Common/StringUtil.h"`